R/percent-time-canada.R

```diff
@@ -0,0 +1,26 @@
+#' Percent time in Canada
+#' 
+#' This function creates a derived variable that provides a percentage of the time a person's life was spent in Canada
+#' 
+#' @param Age_cont continuous age variable. 
+#' cchsflow variable name: DHHGAGE_cont
+#' 
+#' @param BirthCountry whether or not someone was born in Canada 
+#' cchsflow variable name: SDCGCBG
+#' 
+#' @param TimeCanada how long someone has lived in Canada
+#' cchsflow variable name: SDCGRES
+#' 
+#' @return value of percentage of time spent living in Canada in the Pct_time_derived variable
```

Comment:
Returns should all have the value type — usually a number, but could be a factor, text, etc. Percentage as a fraction? (value between 0 and 1)? Please indicate.

Reply:
See comment about parameter names